robot/impl/local_robot.go

```diff
@@ -1587,3 +1594,8 @@ func (r *localRobot) RestartAllowed() bool {
 	}
 	return false
 }
+
+// ListTunnels returns information on available traffic tunnels.
+func (r *localRobot) ListTunnels() []config.TrafficTunnelEndpoint {
+	return r.trafficTunnelEndpoints
```

Comment:
can we use the stored Config instead? so we're not running into/have to worry about running into any caching issues

Reply:
Can do; let me know if what I have now is what you imagined.